models/bridges/base/bridges_base_addresses.sql

```diff
@@ -0,0 +1,19 @@
+{{ config(alias = 'addresses',
+        schema = 'bridges_base',
+        tags=['static'],
+          ) }}
+
+SELECT blockchain, address as address, bridge_name, description
```

Comment:
while there isn't a typo here like the ethereum one, can we remove `address as` and just use `address` without alias since it's the same? only calling it out to prevent copy/paste typo potential in the future to end up like ethereum one below

Reply:
ty. these changes have been implemented